### DISCUSSION TITLE
FooocusAPI shutdown way

### DISCUSSION THREAD

**Discussion by veyakup** [lllyasviel/Fooocus#3646] [Q&A] [answered]
I start the API bat file in python with the code

bat_file_path = r"D:\AI\Fooocus-API\start.bat"
subprocess.run(['start', 'cmd', '/k', bat_file_path], shell=True)

I generate all the images through the API, but when I close the CMD screen, the Fooocus API continues to use the allocated gpu memory on the graphics card in the background.

Is there a way to completely close the FooocusAPI server with the API?

**Comment by mashb1t** [comment, marked as answer]
Wrong repository, this is main Fooocus. Please open a discussion in FooocusAPI.
